yarn.lock

```diff
@@ -2014,11 +2014,16 @@
     prop-types "^15.7.2"
     react-is "^16.8.0"
 
-"@metamask/contract-metadata@^1.19.0", "@metamask/contract-metadata@^1.20.0":
+"@metamask/contract-metadata@^1.19.0":
```

Comment:
Could we dedupe this entry too? This range is compatible with `1.21.0`

Reply:
Done!